products/apache-nifi.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Apache NiFi
+category: server-app
+tags: apache java-runtime
+iconSlug: apachenifi
+permalink: /apache-nifi
+alternate_urls:
+-   /nifi
+changelogTemplate: "https://cwiki.apache.org/confluence/display/NIFI/Release+Notes#ReleaseNotes-Version__LATEST__"
+eolColumn: Support
+releaseDateColumn: true
+
+identifiers:
+-   repology: nifi
+
+auto:
+  methods:
+  -   git: https://github.com/apache/nifi.git
+
+releases:
+-   releaseCycle: "1.27.0"
+    releaseDate: 2024-07-08
+    eol: false
+    latest: "1.27.0"
+-   releaseCycle: "2.0.0-M4"
+    releaseDate: 2024-07-01
+    eol: false
+    latest: "2.0.0-M4"
+-   releaseCycle: "2.0.0-M3"
+    releaseDate: 2024-05-16
+    eol: 2024-07-02
+    latest: "2.0.0-M3"
+-   releaseCycle: "1.26.0"
+    releaseDate: 2024-05-06
+    eol: 2024-07-09
+    latest: "1.26.0"
```

Comment:
```suggestion
    latest: "1.26.0"
    latestReleaseDate: 2024-05-06
```

All releases are missing the `latestReleaseDate` field (can be found on https://github.com/apache/nifi/tags).